docs/07-AdvancedUsage.md

```diff
@@ -664,6 +664,12 @@ groups:
 This will load all found `p*` files in `tests/_data` as groups. Group names will be as follows p1,p2,...,pN.
 
 
+## Shell autocompletion
+For bash and zsh shells, you can use auto completion by executing the following in your shell (or add it to your bash/zshrc):
+````bash
+source <([codecept location] _completion --generate-hook)
```

Comment:
I have tested your change and I got  a problem with `[codecept location]`.

I added `source <( ~/projects/codeception/codecept _completion --generate-hook)` to .bashrc.
Autocompletion only works if I specify an absolute path `/home/user/projects/codeception/codecept`,
it doesn't work for `~/projects/codeception/codecept` and `./codecept`.

I think that it would work better if I had codecept in the PATH and ran `source <(codecept _completion --generate-hook)` instead. But it would be good to make autocompletion work for local composer installation.